cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.container import Container
+from cfme.containers.image import Image
+from cfme.containers.service import Service
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.9"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')
+]
+
+tested_objects = [Container, Image, Service]
+project_name = 'test-project-dashboard'  # Predefine project for this test
```

Comment:
__OPTIONAL__
It's common habit in Python to define constants as UPPERCASE.

Reply:
change constants to UPPERCASE.